src/main/java/com/suse/salt/netapi/calls/modules/State.java

```diff
@@ -28,4 +29,18 @@ private State() { }
         return apply(Arrays.asList(mods));
     }
 
+    public static LocalCall<Map<String, Object>> apply(String mod,
+            Map<String, Object> pillar) {
+        LinkedHashMap args = new LinkedHashMap();
+        args.put("mods", Collections.singletonList(mod));
+        args.put("pillar", pillar);
+        return new LocalCall("state.apply", Optional.empty(), Optional.of(args),
+                new TypeToken<Map<String, Object>>() { });
+    }
```

Comment:
This method currently supports only a single state module, is that right? Why not just adding the `pillar` parameter to the existing apply method so you could give in a list of state modules? Then if we want we can have another method (without the `pillar` parameter) that could just call the other one, if you know what I mean.